docker-compose.confluence.yml

```
@@ -0,0 +1,59 @@
+# Docker Compose configuration for Confluence Integration Testing
+# 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)